.env.example

```diff
@@ -6,9 +6,9 @@ NIL_RPC_ENDPOINT: "http://127.0.0.1:8529"
 
 # Specify the private key used for signing transactions
 # This should be a hexadecimal string corresponding to your account's private key
-PRIVATE_KEY: "41285f03e8692676bf80a98e4052a008026427a7302ca97cb06edcd60689850b"
+PRIVATE_KEY: ""
```

Comment:
Why? I think it is better to show real case in examples

Reply:
We should put an actual private key and wallet address available on the devnet then.
And for local runs the user must create the wallet anyway.
So, I agree with you, but I'm not sure which approach would be better.

Reply:
no big deel now, lets merge 👍

Reply:
Typically env.example contains no values